ap.md

```diff
@@ -0,0 +1,256 @@
+
+</div>
+<div class="large-5 small-12 columns right">
+![router](./images/router.png)
+</div>
+<div class="large-7 small-12 columns">
+Instead of requiring a shared network through a standalone router, we can use the command line tool for configuring Tessel to emit a custom Wifi network and use it as an access point. Then, using Node, we can also start a web server for communicating with any connected device. Unfortunately, we won't be able to connect to the Internet but there is still a lot we can do without that. By the end of this article, we'll be able to control the Tessel LEDs through a web app served by the Tessel.
+</div>
+<div class="large-5 small-12 columns right">
+![router](./images/Tessel-AP.png)
+</div>
+</div>
+
+<div class="row">
+<div class="large-12 columns">
+
+<hr>
+In your command line, make a folder for your Tessel code, then initialize a Tessel project in that folder by running each of the following commands in the terminal:
+
+`mkdir tessel-router`
+
+`cd tessel-router`
+
+`t2 init`
+
+Rename the “index.js” file you’ve just created to “ap.js”, then copy and paste the below script over the existing text:
+
+{% highlight javascript %}
+// Import the interface to Tessel hardware
+var tessel = require('tessel');
+// Load the http module to create an http server.
+var http = require('http');
+
+// Configure our HTTP server to respond with "Hello from Tessel!" to all requests.
+var server = http.createServer(function (request, response) {
+  response.writeHead(200, {"Content-Type": "text/plain"});
+  response.end("Hello from Tessel!\n");
+});
+
+// Listen on port 8080, IP defaults to 192.168.1.101. Also accessible through [tessel-name].local
+server.listen(8080);
+
+// Put a friendly message on the terminal
+console.log("Server running at http://192.168.1.101:8080/");
+
+{% endhighlight %}
+
+Now that our server is in place, let's get our access point set up. In the terminal, run the following command:
+
+`t2 ap -n TesselRouter`
+
+This will make an open, or password-less, wifi network called TesselRouter. If you open the wifi setting of your computer or a separate device like a smartphone or tablet, and scan for new devices, you should be able to see and connect to this new network. For info about making a secure network, take a look at [the cli docs](https://tessel.io/docs/cli#usage). After connecting to TesselRouter, run the following command in your terminal:
+
+`t2 run ap.js`
+
+<div class="small-12 large-6 columns">
```

Comment:
formatting issue - end the previous div before creating this new one, otherwise you get a weird indent